samples/snippets/sample_preemptible.py

```diff
@@ -0,0 +1,148 @@
+#  Copyright 2022 Google LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+# [START compute_preemptible_create]
+import sys
+
+# [END compute_preemptible_create]
+
+# [START compute_preemptible_create]
+# [START compute_preemptible_check]
+# [START compute_preemptible_history]
+from google.cloud import compute_v1
+
+# [END compute_preemptible_history]
+# [END compute_preemptible_check]
+# [END compute_preemptible_create]
+
+
+# [START compute_preemptible_create]
+def create_preemptible_instance(
+    project_id: str, zone: str, instance_name: str,
+) -> compute_v1.Instance:
+    """
+    Send an instance creation request to the Compute Engine API and wait for it to complete.
+
+    Args:
+        project_id: project ID or project number of the Cloud project you want to use.
+        zone: name of the zone you want to use. For example: “us-west3-b”
```

Comment:
```suggestion
        zone: name of the zone you want to use. For example: "us-west3-b"
```
Nit: the quota characters looked odd